spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -256,6 +256,14 @@ def preview_body(body)
       expect(response.content_type).to eq('application/json')
       expect(response).to match_response_schema('autocomplete')
     end
+
+    it 'filter by caution flags' do
+      create(:institution, :count_of_caution_flags, :start_like_harv)
+      get(:autocomplete, params: { term: 'harv' })
+      expect(JSON.parse(response.body)['data'].count).to eq(0)
```

Comment:
You didn't set `exclude_caution_flags` in the `params`.  Shouldn't this then return the institution?

Reply:
Updated it